webdriver/tests/support/helpers.py

```diff
@@ -60,24 +60,37 @@ def _restore_window_state(session):
             session.window.size = defaults.WINDOW_SIZE
 
     @ignore_exceptions
-    def _restore_windows(session):
+    async def _restore_windows(session):
         """Close superfluous windows opened by the test.
 
         It will not end the session implicitly by closing the last window.
+        If available, uses WebDriver BiDi session. Otherwise, uses the WebDriver Classic.
```

Comment:
should it do both in case there is a classic and a bidi session?

Reply:
No need in doing both, as the expected state result is the same, regardless of the protocol used